test/Quake/dependency-if-bug-lifting.qke

```diff
@@ -0,0 +1,124 @@
+// ========================================================================== //
+// Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                 //
+// All rights reserved.                                                       //
+//                                                                            //
+// This source code and the accompanying materials are made available under   //
+// the terms of the Apache License 2.0 which accompanies this distribution.   //
+// ========================================================================== //
+
+// RUN: cudaq-opt --add-wireset --assign-wire-indices --dep-analysis -split-input-file %s | FileCheck %s
+
+// The `quake.h` on `%arg0` will be lifted, causing a conflict as the same qubit will be reused for r and q
```

Comment:
Is this comment out of date? It looks like the `h` is hoisted out of the conditional without an error.

Reply:
It is indeed, the bug the test case show cases has since been fixed :)

Reply:
OK. Let's remove stale comments where we can.

Reply:
The comment has been updated to describe what regression the test is testing for